src/pages/dashboardOverview/DashboardOverview.tsx

```diff
@@ -34,13 +34,14 @@ const DashboardOverview = ({ party, products }: Props) => {
     [party.products]

```

Comment:
You can move the check for ADMIN userRole in activeProducts. In addition to the active products onboarded for the entity, it also returns those that the latter can actually manage. I would give a more appropriate name to the constant such as **manageablePartyProducts**